src/util/patientUtils.js

```diff
@@ -93,6 +93,9 @@ const retrievePatientIds = async ({ subject, organization }) => {
     return referencedObject.member.map(m => m.entity.reference.split('/')[1]);
   } else if (reference[0] === 'Patient') {
     return [subject.split('/')[1]];
+  } else if (reference[0] === 'Practitioner') {
```

Comment:
I think you're missing some logic here. Reference will never equal practitioner since the split statement on 84 can only be subject or organization. Also, since practitioner can only be provided with an organization, this conditional should be inside the organization conditional, and the query should check that the patients generalPractitioner is equal to practitioner and that the managingOrganization is equal to organization